source/pyspark_functions/tests/test_data_frame_wrapper.py

```diff
@@ -0,0 +1,100 @@
+from pyspark.sql.types import StructType, StructField, StringType, IntegerType, DoubleType
+from pyspark_functions.data_frame_wrapper import DataFrameWrapper
+import pytest
+
+
+def test__data_frame_wrapper_initialization(spark):
```

Comment:
we normally use this pattern in test names:
def test__<subject_under_test>__when_<some_when_condition>__<expected_behavior>

The <subject_under_test> and the when-part is optional. 

In this case all tests use data_frame_wrapper, so I would probably just do like this: 
def test__ctor__when_valid_input__returns_expected_schema_and_data

Reply:
similar comment to the other test names